media-video/ffmpeg/ffmpeg-9999.ebuild

```diff
@@ -419,6 +420,20 @@ multilib_src_configure() {
 		"${myconf[@]}"
 	echo "${@}"
 	"${@}" || die
+
+	if multilib_is_native_abi && use chromium; then
+		einfo "Configuring for Chromium"
+		mkdir -p ../chromium || die
+		cd ../chromium || die
```

Comment:
please use pushd/popd here; it'd make it simpler if we want to add more stuff below later